src/cls/rbd/cls_rbd.cc

```diff
@@ -132,6 +133,7 @@ cls_method_handle_t h_mirror_image_status_get;
 cls_method_handle_t h_mirror_image_status_list;
 cls_method_handle_t h_mirror_image_status_get_summary;
 cls_method_handle_t h_mirror_image_status_remove_down;
+cls_method_handle_t h_create_cg;
```

Comment:
Since we are going to have a bunch of consistency group-related methods, can we name them like "cg_[optional-subtype}_]{verb}" format?  e.g. cg_create, cg_dir_list, etc